dali/base/dasess.cpp

```diff
@@ -2012,9 +2013,11 @@ class CUserDescriptor: implements IUserDescriptor, public CInterface
     StringAttr passwordenc;
     unsigned sessionToken;//ESP session token
     StringBuffer signature;//user's digital Signature
+    IDigitalSignatureManager * pDSM;
 public:
     IMPLEMENT_IINTERFACE;
     CUserDescriptor()
+        : pDSM(nullptr)
```

Comment:
You may set nullptr at line 2016: IDigitalSignatureManager *pDSM = nullptr;

Reply:
Class members have to be initialized in a method.  RKC informed me that the pattern is to use the member initialization list in the CTOR

Reply:
I think preference now is to initialize members at declaration time for a cleaner look and avoid repetition if multiple CTOR's.
pDSM and sessionToken initialization can be be move there.

Mentioning @richardkchapman in case some other pattern he meant.

Reply:
@jakesmith Just last week RKC had me change something I was initializing in the CTOR and move it to the member initialization list.  @richardkchapman Please advise

Reply:
>I was initializing in the body of the CTOR and move it to the member initialization list.

I am not suggesting you put in the CTOR body.

Reply:
I know you are not saying to put it in the body. I am saying that Richard told me last week to use the initialization list, although I see the advantage of initializing where declared.